assignments/callbacks.js

```diff
@@ -2,29 +2,75 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function firstItem(arr, cb) {
   // firstItem passes the first item of the given array to the callback function.
+  cb(arr);
 }
```

Comment:
instructions want firstItem to do the work here, so it would be `cb(arr[0])`, and the cb would just console.log whatever it's given